src/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM centos:centos8.3.2011
```

Comment:
What env will this run on? From the looks of it `FROM: scratch` doesn't seem to be an option but maybe using an rhel container is? Maybe something with a `slim` flavour?

Reply:
This is suggested by huawei itself.